tests/src/lib_tests.rs

```diff
@@ -1153,6 +1153,19 @@ test_contract_call_response!(
     }
 );
 
+test_contract_call_response!(
```

Comment:
Heads up, we're migrating to a new unit test model where the tests are directly in the Clarity code. This is fine for now though.